### ISSUE TITLE
Deprecation warning for distutils in SentenceTransformer.py line 22 (Ubuntu 22.04)

### ISSUE THREAD

**Issue by gruffdavies** [UKPLab/sentence-transformers#2394]
sentence-transformers==2.2.2

```
.venv/lib/python3.10/site-packages/sentence_transformers/SentenceTransformer.py:22: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.dir_util import copy_tree
```

**Comment by bwanglzu** [comment]
it's interesting that this has been imported while never being used in the source.

**Comment by tomaarsen** [comment]
Hello!

Thanks for raising! Will be solved by #2399

- Tom Aarsen
